lib/handlers_v5.js

```diff
@@ -341,3 +344,21 @@ function shutdown_request(request) {
 
     status_idle.call(this, request);
 }
+
+function is_complete_request(request) {
```

Comment:
Eventually, the parser would have to be moved into `jp-babel`, `jp-javascript`, and `jp-coffeescript` (I want to keep `jp-kernel` kernel agnostic).

I want to refactor the way Kernel implements the protocol message handlers and make it more OOP. I'm also unhappy with the names `Kernel#startupCallback` and `Kernel#startupScript`. Once I'm done with refactoring Kernel, I'll revisit how the parser is handled.